src/components/sub-components/mgt-flyout/mgt-flyout.ts

```diff
@@ -27,33 +28,88 @@ export class MgtFlyout extends LitElement {
     return styles;
   }
 
+  /**
+   * Gets or sets whether the flyout is light dismissable.
+   *
+   * @type {boolean}
+   * @memberof MgtFlyout
+   */
+  @property({
+    attribute: 'light-dismiss',
+    type: Boolean
+  })
+  public isLightDismiss: boolean;
+
   /**
    * Gets or sets whether the flyout is visible
    *
    * @type {string}
-   * @memberof MgtComponent
+   * @memberof MgtFlyout
    */
   @property({
     attribute: 'isOpen',
     type: Boolean
   })
-  public isOpen: boolean;
+  public get isOpen(): boolean {
+    return this._isOpen;
+  }
+  public set isOpen(value: boolean) {
+    if (this._isOpen !== value) {
+      this._isOpen = value;
+
+      if (value) {
+        this.registerLightDismissEvents();
+      } else {
+        this.unregisterLightDismissEvents();
+      }
 
-  private renderedOnce = false;
+      this.requestUpdate('isOpen');
+
+      if (!value) {
+        this.fireCustomEvent('onclose');
```

Comment:
the event should be called `close` (or `closed`) and there should be event `open` (or `opened`) for consistency

Reply:
Good idea